application/pages/modplatform/twitch/TwitchModel.cpp

```diff
@@ -263,11 +263,12 @@ void Twitch::ListModel::searchRequestFinished()
             }
             pack.latestFile.addonId = pack.addonId;
             pack.latestFile.fileId = id;
-            // FIXME: what to do when there's more than one, or there's no version?
             auto versionArray = file.value("gameVersion").toArray();
-            if(versionArray.size() != 1) {
+            if(versionArray.size() > 0) {
```

Comment:
This change is super weird. Are you sure this is correct?

This seems like exactly the opposite of what it should be.

Reply:
I messed up the direction when moving this from my Linux VM to my host PC to commit it, will fix and rebase.